requirements.txt

```diff
@@ -5,6 +5,8 @@ Pattern
 wordcloud
 pygal
 nose
-psycopg2
-
-
+psycopg2==2.7.1
```

Comment:
remove?

Reply:
It is needed. It is used by sqlalchemy to support features specific to postgresql. It's called a 'dialect' of sql in sqlalchemy terms.

Reply:
ok. perfect

Reply:
ok.. got it